nix/nft-marketplace-frontend.nix

```diff
@@ -0,0 +1,38 @@
+{ config, lib, pkgs, ... }:
```

Comment:
i'm not even sure this should be a separate module. maybe we can merge it with `nft-marketplace-server`